tests/support/DbHelper.php

```
@@ -34,4 +34,38 @@ public static function dropTablesIfExist(Connection $db, array $tables): void
         }
     }
+
+    /**
+     * Creates the given schema on SQL Server when it does not already exist.
+     *
+     * Targets the MSSQL driver through `SCHEMA_ID()` and a batched `EXEC('CREATE SCHEMA ...')` statement.
+     *
+     * @param Connection $db Database connection.
+     * @param string $schema Schema name to create.
+     */
+    public static function createSchemaIfNotExist(Connection $db, string $schema): void
```

PHPMD (2.15.0)

[warning] Avoid variables with short names like $db. Configured minimum length is 3. (undefined)

(ShortVariable)